### PR TITLE
NPC can now attack, player loose HP

### PR DESCRIPTION
**Pull request by dderancourt**

### CODE REVIEW

**Review by TheoBoschetti — DISMISSED** [review]
ok

**Review by TheoBoschetti — DISMISSED** [review]
ok

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
- `pahtfinder.js` ? :p 
- `var` ?!
- `console.log` is forbidden, I want an empty console!

**Review by TheoBoschetti — DISMISSED** [review]
ok

**Review by TheoBoschetti — DISMISSED** [review]
ok

File: frontend/src/Components/Game.jsx [hunk 1]
```diff
@@ -314,6 +347,8 @@ class Game extends React.Component {
     this.getSword();
   }
 
+  playBounce() {}
```
Comment (LoicBrassart) [inline frontend/src/Components/Game.jsx]: Nope!

**Review by LoicBrassart — CHANGES_REQUESTED** [review]

**Review by TheoBoschetti — APPROVED** [review]
ok

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**